README.md

```diff
@@ -98,6 +98,15 @@ on which the Django project is running (service "web"), run `make bash-shell`.
 * In order to access the Django shell, run `make django-shell`.
 * In order to access the mysql shell, run `make db-shell`.
 
+### Writing to/reading from the host machine
+If you need to write out data from a local impact-api instance to your host
+machine's drive, the directory `/wwwroot` on the docker 
+container is mapped to `impact-api/web/impact` on your host machine. If you are
+working in a django shell, creating a filehandle `f = open("/wwwroot/foo.txt", "w")`
+will allow you to write out data to your host machine. 
```

Comment:
`s/to your/that can also be seen from your/`

Reply:
@pbx back to you